### PR TITLE
ODM-11311: Skos documentation

### PR DESCRIPTION
**Pull request by viapivov**

### CODE REVIEW

File: docs/user-guide/dictionaries-and-ontologies/loading-new-ontology/skos-dictionary.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Dictionaries in SKOS namespace
+
+ODM also supports dictionaries in [SKOS](https://www.w3.org/2009/08/skos-reference/skos.html) namespace. This feature is
+useful when you
+want to organize terms in a hierarchical structure, which enables more advanced search capabilities. Both RDF/XML and
+TURTLE formats
+of dictionaries are supported. Predicated `skos:broader`, `skos:narrower` `skos:related`, `skos:definition`,
```
Comment (m-smazhevsky) [inline docs/user-guide/dictionaries-and-ontologies/loading-new-ontology/skos-dictionary.md]: Predicate**S**?

File: docs/user-guide/dictionaries-and-ontologies/loading-new-ontology/skos-dictionary.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	
+<http://vocabulary.boehringer-ingelheim.com/BodySystem/104>
+	a skos:Concept ;
+	skos:narrower <http://vocabulary.boehringer-ingelheim.com/BodySystem/562> ;
+	skos:prefLabel "Brain"@en .
+	
+<http://vocabulary.boehringer-ingelheim.com/BodySystem/562> skos:broader <http://vocabulary.boehringer-ingelheim.com/BodySystem/104> .
+```
+</details>
+
```
Comment (m-smazhevsky) [inline docs/user-guide/dictionaries-and-ontologies/loading-new-ontology/skos-dictionary.md:31]: And here is an example of the same dictionary but in RDF/XML format:

**Review by m-smazhevsky — CHANGES_REQUESTED** [review]
Excelent!

File: docs/user-guide/dictionaries-and-ontologies/loading-new-ontology/skos-dictionary.md (hunk 1 above)
Comment (viapivov) [reply docs/user-guide/dictionaries-and-ontologies/loading-new-ontology/skos-dictionary.md]: Yes, thanks

File: docs/user-guide/dictionaries-and-ontologies/loading-new-ontology/skos-dictionary.md (hunk 2 above)
Comment (viapivov) [reply docs/user-guide/dictionaries-and-ontologies/loading-new-ontology/skos-dictionary.md:31]: Slightly rephrased

**Review by genestack-okunitsyn — DISMISSED** [review]

File: docs/user-guide/dictionaries-and-ontologies/loading-new-ontology/skos-dictionary.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+# Dictionaries in SKOS namespace
+
+ODM also supports dictionaries in [SKOS](https://www.w3.org/2009/08/skos-reference/skos.html) namespace. This feature is
+useful when you
```
Comment (MikhailAf) [inline docs/user-guide/dictionaries-and-ontologies/loading-new-ontology/skos-dictionary.md]: leave just 'useful to organise' and organise with 's'

File: docs/user-guide/dictionaries-and-ontologies/loading-new-ontology/skos-dictionary.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+want to organize terms in a hierarchical structure, which enables more advanced search capabilities. Both RDF/XML and
+TURTLE formats of dictionaries are supported. Predicates `skos:broader`, `skos:narrower` `skos:related`,
+`skos:definition`, and `skos:exactMatch` are recognized and utilized by ODM.
```
Comment (MikhailAf) [inline docs/user-guide/dictionaries-and-ontologies/loading-new-ontology/skos-dictionary.md:6]: recogniSed and utiliSed

File: docs/user-guide/dictionaries-and-ontologies/loading-new-ontology/skos-dictionary.md (hunk 3 above)
Comment (viapivov) [reply docs/user-guide/dictionaries-and-ontologies/loading-new-ontology/skos-dictionary.md]: It's beyond repair here

File: docs/user-guide/dictionaries-and-ontologies/loading-new-ontology/skos-dictionary.md (hunk 4 above)
Comment (viapivov) [reply docs/user-guide/dictionaries-and-ontologies/loading-new-ontology/skos-dictionary.md:6]: It's beyond repair here

File: docs/user-guide/dictionaries-and-ontologies/loading-new-ontology/skos-dictionary.md (hunk 3 above)
Comment (viapivov) [reply docs/user-guide/dictionaries-and-ontologies/loading-new-ontology/skos-dictionary.md]: fixed, but not s->z

**Review by MikhailAf — APPROVED** [review]

**Review by m-smazhevsky — APPROVED** [review]

**Pull request merged**